src/backend/expungeservice/models/charge_types/duii.py

```diff
@@ -9,17 +10,18 @@ def _default_type_eligibility(self):
         if not self.disposition:
             return TypeEligibility(EligibilityStatus.NEEDS_MORE_ANALYSIS, reason='Further Analysis Needed - No disposition')
 
-        elif 'diverted' in self.disposition.ruling.lower():
-            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason='Ineligible under 137.225(8)(b)')
+        elif self.disposition.status == DispositionStatus.DIVERTED:
+            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason='137.225(8)(b) - Diverted DUII charges are ineligible')
 
-        elif self.acquitted():
-            return TypeEligibility(EligibilityStatus.NEEDS_MORE_ANALYSIS, reason='Further Analysis Needed')
+        elif self.convicted():
 
-        else:
             """
-            TODO: other dispositions than "convicted" are possible and are not handled here.
-            See issue: https://github.com/codeforpdx/recordexpungPDX/issues/601
-
             TODO: reason can be expanded to "137.225(7)(a) - Traffic offenses are ineligible"
             """
-            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason='Ineligible under 137.225(7)(a)')
+            return TypeEligibility(EligibilityStatus.INELIGIBLE, reason='137.225(7)(a) - Traffic offenses are ineligible')
+
+        elif self.acquitted():
```

Comment:
I think this case requires more of a comment as the naive reader may otherwise rebut that "wait a minute, isn't the diverted case handled above"?